tests/nvme_copy_test.py

```diff
@@ -1,17 +1,19 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
nvme-cli is license under GPL-2.0-only, libnvme is LGPL-2.1-or-later. I don't mind if you license this file under LGPL as it is compatible.